endpoints-support.md

```diff
@@ -14,6 +14,14 @@ Refer SimpliVity REST API doc for the resource endpoints documentation [HPE Simp
 |<sub>/backups/{bkpId}/lock</sub>                                  |POST    |
 |<sub>/backups/{bkpId}/rename</sub>                                |POST    |
 |<sub>/backups/{bkpId}/restore</sub>                               |POST    |
+|<sub>/backups	</sub>                                                                    |GET       |
```

Comment:
remove duplicate entries

Reply:
Removed the duplicate entries and corrected the sequence.